docs/api.rst

```diff
@@ -43,13 +43,21 @@ SchemaGenerator
       in the schema. When the handler is executed, the request body will be read and
       validated against that type.
 
-   .. method:: api_view()
+      :param tags: Iterable object used to specify tags to group endpoints.
```

Comment:
It would be nice if you add a mention that tags are strings. They are strings, right?

Reply:
Yep, they are